include/aspect/utilities.h

```diff
@@ -1064,6 +1065,41 @@ namespace aspect
          */
         const std::function<Tensor<1,dim> (const Point<dim> &)> function_object;
     };
+
+    /**
+     * Create a permutation vector which can be used by the apply_permutation function
+     * which shorts the vector in increasing order.
```

Comment:
```suggestion
     * which sorts the vector in increasing order.
```
?
Can you make the function description more precise as well?